Gopkg.toml

```diff
@@ -18,6 +18,10 @@ required = [
   branch = "master"
   name = "github.com/cupcake/jsonschema"
 
+[[override]]
+  name = "github.com/docker/docker"
+  branch = "master"
+
```

Comment:
Is this override necessary? I thought we dropped the docker dependency?

Reply:
Removed, thanks.